lib/virtfs/xfs/file_class.rb

```diff
@@ -0,0 +1,224 @@
+module VirtFS::XFS
+  class FS
```

Comment:
@roliveri the suggestion above by @Fryguy above makes perfect sense.  Is this an ok way to go from your perspective?